vendor/assets/javascripts/react-chosen.js

```diff
@@ -0,0 +1,69 @@
+(function(window, React, $) {
```

Comment:
`RailsAssets.load_paths` is only for non-rails apps I think. 

Of course we can manually rip things out of the load path. We can even put it into an initializer in the railtie of ui_c (please not in the dummy app, then we have to copy&paste that into every project!). It's not pretty but if there is no other way...